tests/test_scene_manager.py

```diff
@@ -94,7 +94,8 @@ def test_save_images(test_video_file):
     sm.add_detector(ContentDetector())
 
     image_name_glob = 'scenedetect.tempfile.*.jpg'
-    image_name_template = 'scenedetect.tempfile.$SCENE_NUMBER.$IMAGE_NUMBER'
```

Comment:
Could you remove the whitespace present in the resulting string?  You can use string concatenation if you place the expression in brackets:
    
    image_name_template = ('scenedetect.tempfile.'
                           '$SCENE_NUMBER.$IMAGE_NUMBER.$FRAME_NUMBER.$TIMESTAMP_MS.$TIMECODE')

Thank you for updating the unit tests!